### PR TITLE
Update to Screenshot.md

### PR DESCRIPTION
**Pull request by peterleimbach**
1) updated home screen screenshot to reflect the 4 buttons at the bottom.
2) described 3 of the 4 dialogs which can be called via the buttons at the bottom. The calculator war shortly described already.

### CODE REVIEW

**Comment by peterleimbach** [comment]
Hi Adrian,

I changed the PR considering your comments.

Best Regards
Peter

File: docs/EN/Getting-Started/Screenshots.md [hunk 1]
```diff
@@ -31,7 +31,19 @@ The solid blue line shows the basal delivery of your pump. The dotted blue line
...
+
+## Insulin
+The Insulin dialog gives you the possibility to give insulin. In addition you can set a temp target e.g. because you plan to eat and want to decrease your blood sugar level below your normal blood sugar target or just push more aggressive down. This can help to decrease the spike after meal. Furthermore you can inform AAPS about insulin which you gave outside of AAPS e.g. with a pen. This is necessary because otherwise the IOB calculation would be inaccurate.
+For further information about temp targets please look at [Temp Targets (TT)](../Usage/temptarget.html#temp-targets).
+
+![Insulin Dialog](../images/Screenshot_Insulin_Dialog.png)
+
+## Carbs
+The Carbs dialog gives you the possibility to inform AAPS about Carbs you eat. In addition you can use the Carbs dialog to increase your temp target for a activty or to handle a hypo. This option is pre-selected if your blood sugar is below 4 4 mmol/l (72 mg/dl). You can set your defautl targets and times via Preferences->Other->Default Temp-Targets. The last thing you can do is for handling of eCarbs. The concept is described at [Extended carbs / “eCarbs”](../Usage/Extended-Carbs.html#extended-carbs-ecarbs).
```
Comment (AdrianLxM) [inline docs/EN/Getting-Started/Screenshots.md]: Typo: "a activity" -> "an activity"

Comment (AdrianLxM) [reply docs/EN/Getting-Started/Screenshots.md]: I believe the 4 of 4mmol/l is duplicated

File: docs/EN/Getting-Started/Screenshots.md [hunk 2]
```diff
@@ -47,6 +59,11 @@ SUPER BOLUS is where the basal insulin for the next two hours is added to the im
 
 **Section C:** shows the various elements that have been used to calculate the bolus. You can deselect any that you do not want to include but you normally wouldn't want to.
 
+## CGM calibration
+The CGM calibration dialog is for input of a blood sugar which you need to calibrate your CGM or FGM.
+
+![VGM calibration Dialog](../images/Screenshot_Calibration_Dialog.png)
```
Comment (AdrianLxM) [inline docs/EN/Getting-Started/Screenshots.md]: "VGM" -> "CGM"

**Pull request closed without merging**